pages/index.js

```diff
@@ -30,9 +35,15 @@ export default function Home() {
 
       <Main>
         <div>
```

Comment:
whenever you map over something you are creating multiple elements of the same "type" or structure. In these cases we normally want to express them as "li" elements inside an "ul". Please change the elements of this wrapper and of the created Cards to fit this pattern.